### PR TITLE
Initial Django Install and project configuration

### PR DESCRIPTION
**Pull request by lili2311**
### ISSUE/TICKET NUMBER
[3](https://trello.com/c/PoIEOvkX/24-task-3-install-the-initial-project)

### Describe the changes
Initial install of django with custom user model

### CODE REVIEW

**Comment by raquel-ucl** [comment]
I know this PR doesn't include the README, but since it's about project configuration, while setting up I've noticed there's no requirements.txt file yet. Should we add one? Otherwise needs correction in the README file.

File: README.md [hunk 1]
```diff
@@ -33,6 +35,91 @@ A collaborative, youth­led project aiming to make use of digital technology to
...
+	- Homebrew ```brew install postgresql```
+	- OR Download the [Postgress App](http://postgresapp.com/)
+
+9. Make sure the Postgres Server is up and running:
+	- If using the App simply start the server from there
+	- If using command line: ``brew services start postgresql``
+
+9. Install the requirements:
+
+	 ```pip install -r requirements.txt```
```
Comment (raquel-ucl) [inline README.md]: Can't find a requirements.txt file in this branch. Forgot to add?

File: README.md [hunk 2]
```diff
@@ -33,6 +35,91 @@ A collaborative, youth­led project aiming to make use of digital technology to
...
+	 
+	 This will get all the dependencies.
+
+9. Create a database locally for the project to run:
+	
+	```createdb ocdaction```
+
+10. Go inside frontend folder: 
+	
+	```cd frontend```
```
Comment (raquel-ucl) [inline README.md]: This should be `cd ocdaction/frontend`.

File: README.md (hunk 1 above)
Comment (lili2311) [inline README.md]: strange, yep not here will add it in!

File: README.md (hunk 2 above)
Comment (lili2311) [inline README.md]: also strange i think some files are missing in this PR, il go back and see what happened and push them all.

File: README.md [hunk 3]
```diff
@@ -33,6 +35,91 @@ A collaborative, youth­led project aiming to make use of digital technology to
...
+	 This will get all the dependencies
+
+	
+## Running the project locally
+1. Go inside the django app directory: 
+
+	```cd ocdaction```
+2. Run django server:
+	
+	```python manage.py runserver```
```
Comment (raquel-ucl) [inline README.md]: I get `No module named ocdaction.settings`, although I can see the settings file there. Maybe it's my config? I'll check.

Comment (lili2311) [inline README.md]: pushed some missing files can you try again?

**Comment by lili2311** [comment]
@raquel-ucl please advise if this is working for you now?

**Pull request merged**